include/PROPOSAL/geometry/Box.h

```diff
@@ -38,7 +38,7 @@ namespace PROPOSAL {
 class Box : public Geometry
 {
 public:
-    Box(const Vector3D position, double x, double y, double z);
+    Box(const Vector3D& position, double x, double y, double z);
```

Comment:
shouldn't this be Cartesian3D?

Reply:
The user may want to describe the position of a Box in spherical coordinates, in this case he could pass a Spherical3D object. The concept is probably more useful in cases where we are talking about directions, but nothing keeps you from describing positions in spherical coordinates as well 😉.